lib/web/integrations_awsra.go

```diff
@@ -145,6 +145,56 @@ func (h *Handler) awsRolesAnywhereConfigureTrustAnchor(w http.ResponseWriter, r
 	return nil, trace.Wrap(err)
 }
 
+// awsRolesAnywherePing performs an health check for the integration.
+// It returns the caller identity and the number of AWS Roles Anywhere Profiles that are active.
+// If a trust anchor is provided in the body, it will be used to check the connection ignoring the integration.
+// Otherwise, the integration is used to check the connection.
+func (h *Handler) awsRolesAnywherePing(w http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (any, error) {
+	ctx := r.Context()
+
+	integrationName := p.ByName("name")
+	if integrationName == "" {
+		return nil, trace.BadParameter("an integration name is required")
+	}
+
+	var req ui.AWSRolesAnywherePingRequest
+	if err := httplib.ReadJSON(r, &req); err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	clt, err := sctx.GetUserClient(ctx, site)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	// When creating an integration, the Ping is called with an empty integration, but Trust Anchor, Profile and Role ARNs must be provided.
+	// This allow us to check if the integration is properly configured before creating it.
+	if req.TrustAnchorARN != "" {
+		integrationName = ""
+	}
```

Comment:
isn't `integrationName` required a few lines above? should we just make two separate APIs?

Reply:
It is required but not actually used when testing either the first creation or a change in the configuration fields.
For those situations we require it because it is a part of the URL path.

I'm trying to mimic what we already have for the AWS OIDC Ping endpoint.
At this point, we already have the integration name anyway in both scenarios.